can't build in ~/.local in Fedora Silverblue

Trying to install `apx` in Silverblue locally. But it is asking for sudo access.  
```
git clone --recursive https://github.com/Vanilla-OS/apx.git
cd apx
make build PREFIX=$HOME/.local
make install PREFIX=$HOME/.local
```
error
```
install -Dm755 apx //home/zihad/.local/bin/apx
mkdir -p //etc/apx
mkdir: cannot create directory ‘//etc/apx’: Permission denied
make: *** [Makefile:12: install] Error 1
```

any solution? Why it needs access to /etc/apx. Can't it be build in `~/.local`?

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ build:
 
 install:
 	install -Dm755 ${BINARY_NAME} ${DESTDIR}${PREFIX}/bin/${BINARY_NAME}
-	mkdir -p ${DESTDIR}/etc/apx
+	sudo mkdir -p ${DESTDIR}/etc/apx
 	sed -i 's|/usr/share/apx/distrobox|${PREFIX}/share/apx/distrobox|g' config/config.json
 	sudo install -Dm644 config/config.json ${DESTDIR}/etc/apx/config.json
 	mkdir -p ${DESTDIR}${PREFIX}/share/apx

